tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Models/Matter/MatterConfigurations.cs

```diff
@@ -232,7 +232,12 @@ public IList<string> UserId
         /// <summary>
         /// Property to determine the matter type(Guid or DateTime of Custom
         /// </summary>
-        public string MatterIdType { get; set; } 
+        public string MatterIdType { get; set; }
+
+        /// <summary>
+        /// Property to determine the matter type(Guid or DateTime of Custom
+        /// </summary>
+        public IList<MatterExtraFields> AdditionalFieldValues { get; set; }

```

Comment:
Adding List of fields so we can push fields object while creating matter at assign content type step.